src/state/urls.js

```diff
@@ -16,3 +16,17 @@ export function getBigQueryURL(appName, appId, pingName, metricName) {
 
   return base + (metricName ? `?search=${metricName}` : "");
 }
+
+export function getLookerURL(appName, pingName) {
+  // this is currently only supported for a hardcoded set of app ids and is a bit of a
+  // hack (we should ideally be doing this somewhere in our upstream ETL)
+
+  if (appName === "fenix" || appName === "burnham") {
+    return `https://mozilladev.cloud.looker.com/explore/${appName}/${pingName.replace(
```

Comment:
Should change this when fenix lands on production.

Reply:
I don't have access to this dev site. is it restricted to only the people who are working on Looker?

Reply:
Yes, though that will almost certainly include you in the near future-- I suspect we should add you to the list of developers for Looker.

Reply:
This should be prod now.